### PR TITLE
Remove duplicate entries in BlockInfo tab of worker WEBUI

### LINKED ISSUE #9518: Display three the same records in  BlockInfo tab of worker node WEBUI
**Alluxio Version:**
2.1.0-SNAPSHOT

**Describe the bug**
when entering page of **BlockInfo** of worker node , each file has three the same records in WEBUI.
clicked the link of file which three the same records, display the same information in WEBUI.
图片

图片

**To Reproduce**
enter page of BlockInfo of worker node 

**Expected behavior**

**Urgency**
Describe the impact and urgency of the bug.

**Additional context**

### PR DESCRIPTION
**Pull request by tieujason330**
- Fixes #9518 
- following up on style fixes for #9577

### CODE REVIEW

**Comment by tieujason330** [comment]
@calvinjia PTAL, thanks

File: core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java [hunk 1]
```diff
@@ -343,6 +343,8 @@ public Response getWebUIBlockInfo(@QueryParam("path") String requestPath,
                 uiFileInfo
                     .addBlock(blockMeta.getBlockLocation().tierAlias(), blockId, blockSize, -1);
               }
+            }
+            if(uiFileInfo.getBlockIds().isEmpty()) {
```
Comment (calvinjia) [inline core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java]: spacing

Comment (phenixmzy) [reply core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java]: oh，I am very sorry. when i commit fix forgot add the 
**private final List<Long> mBlockIds = new ArrayList<>();**
to branch.
i will pay attention to it and add a unit test next time.

**Comment by calvinjia** [comment]
@tieujason330 could you address the checkstyle issues please?

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Pull request merged**